app/src/main/java/org/p2p/wallet/debug/settings/DebugSettingsFragment.kt

```diff
@@ -7,7 +7,7 @@ import org.koin.android.ext.android.inject
 import org.p2p.wallet.R
 import org.p2p.wallet.common.mvp.BaseMvpFragment
 import org.p2p.wallet.databinding.FragmentDebugSettingsBinding
-import org.p2p.wallet.debug.featuretoggles.FeatureTogglesFragment
```

Comment:
do not like _ in packages

Reply:
hm, i thought we have set naming in our code style, but i was wrong.
should we use Kotlin `camelCase.packageName` style then?

Reply:
@1sakov ^^ ?
i prefer camelCase if this won't harm our CI for example

Reply:
I'd leave as it is. 
We have several similar folders already. like pushnotifications

So, let's leave as `featuretoggles`

Reply:
done